niworkflows/tests/conftest.py

```diff
@@ -27,12 +31,6 @@ def _run_interface_mock(objekt, runtime):
     return runtime
 
 
-# XXX This may not be necessary. Look into more when overhauling tests.
-def pytest_runtest_setup(item):
-    """Change to temporary directory"""
-    os.chdir(mkdtemp())
```

Comment:
Removed this as a potential cause of the `pytest-cov` issue. Individually `chdir`ing tests that need it feels cleaner.

This has led to my repo getting polluted, so I'll track down what's causing that and `chdir` or adjust paths as needed.